### DISCUSSION TITLE
the image showing venerabilties related to glibc

### DISCUSSION THREAD

**Discussion by sushantgoswami** [oracle/container-images#54] [Q&A]
the image need ksplice package updates, glibc, glibc-common etc need fixing.

![image](https://user-images.githubusercontent.com/38786824/214010539-2cd00fc9-b49f-41ac-b6d3-d0d68c2c2b37.png)

**Comment by tvierling** [comment]
This is a false positive. Your security scanner is demanding that Ksplice Userspace packages be installed in the image, but those are not supported within Oracle Linux container images.

Please report this as a false positive to your scanner vendor. We have already reported this issue to both Trivy and Anchore in particular.
